src/core/parser.ml

```diff
@@ -564,6 +564,7 @@ let parser config =
       sanity_check _loc_id id;
       Prefix.add declared_ids id id;
       P_config_ident(id)
+  | "unif_rule" h:rule -> P_config_unif_rule(h)
```

Comment:
I actually preferred the previous syntax, even though it required more code. The problem here is that you would need to declare infix symbols to get what you had before. That is annoying. Better have some specific scoping handling I think.

Reply:
Me too, I'll come back to it when we are settled.